releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,13 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v1.4.4 - July, 22 2021
+* Added the capability to use certificate based authentication.
+* Added logs that show more details with reference to parameters used to run the task.
+* The Azure App Configuration Push pipeline task is now generally available..

```

Comment:
There is an extra period at the end of the sentence.

Can we move this to the first of the list?